lib/api.js

```diff
@@ -37,61 +35,15 @@ API.prototype.defaults = {
 
 API.prototype.start = function() {
   this.createServer();
-  this.listen();
 };
 
 API.prototype.createServer = function createServer() {
-  this.express = this._express();
-  this.server = this.http = this._http();
   this.mqtt = this._newMqtt();
   this.mqtt.start();
 
-  this.express.use(function(req, res, next) {
-    res.header('Access-Control-Allow-Origin', this.CORS || '*:*');
-    res.header('Access-Control-Allow-Headers',
-               'Origin, Authorization, X-Requested-With, Content-Type, Accept');
-    next();
-  }.bind(this));
-
-  this.express.set('title', 'Cylon MQTT API');
-
-  this.express.get('/', function(req, res){
-    res.sendFile(__dirname + '/index.html');
-  });
-
-  // error handling
-  this.express.use(function(req, res, next) {
-    if (req.err) {
-      res.status(500);
-      res.json({ error: req.err.message || 'An error occured.' });
-      next(req.err);
-    } else {
-      next();
-    }
-  });
-};
-
-API.prototype.listen = function() {
-  this.server.listen(this.port, function() {
-    var title = this.express.get('title'),
-        str;
-
-    str = 'Listening ' + title + ' @' + this.host + ':' + this.port;
-
-    console.log('Cylon + MQTT is now publishing.');
-    console.log(str);
-
-  }.bind(this));
+  console.log('Cylon + MQTT is now publishing.');
```

Comment:
Shouldn't we use `Cylon.logger`? Also, a better message might be be "cylon-api-mqtt ready" or something like that.